src/main/java/ConversationManager.java

```diff
@@ -0,0 +1,91 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class ConversationManager {
+    private static final String LINE_SEPERATOR = "------------------------------";
+    private static final String RESPONSE_INDENTATION = "    ";
+    private static final Scanner sc = new Scanner(System.in);
+
+    public static final List<String[]> getUserInput() {
+        String[] inputs = sc.nextLine().split(" ");
+        String[] command = Arrays.copyOfRange(inputs, 0, 1);
+        String[] args = Arrays.copyOfRange(inputs, 1, inputs.length);
+        return List.of(command, args);
+    }
+
+    /**
+     * Prints an intial greeting.
+     */
+    public static final void intializeConversation() {
+        System.out.println(LINE_SEPERATOR);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(LINE_SEPERATOR);
+    }
+
+    /**
+     * Prints a farewell greeting.
+     */
+    public static final void exitConversation() {
+        System.out.println(LINE_SEPERATOR);
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(LINE_SEPERATOR);
+        sc.close();
+    }
+
+    /**
+     * Prints a response with proper indentation
+     * and separators between lines.
+     * Note that RESPONSE_INDENTATION and LINE_SEPARATOR are class-level constants
+     * defined above
+     * 
+     * @param messages a list of string
+     */
+    public static final void printNormalResponse(List<String> messages) {
```

Comment:
(Suggestion) Instead of repeatedly do System.out.println(RESPONSE_INDENTATION + LINE_SEPERATOR); for each type of response, you an declare a function 
```suggestion
    public static final void printResponse(String message) {
        System.out.println(RESPONSE_INDENTATION + LINE_SEPERATOR);
        System.out.println(message);
        System.out.println(RESPONSE_INDENTATION + LINE_SEPERATOR);
    }
    
    public static final void printNormalResponse(List<String> messages) {
        String message = "...";  // generate your message from your list 
        printResponse(message);
    }
```